local/assets/ignition/etcd.yml

```diff
@@ -14,12 +14,13 @@ systemd:
             Environment="ETCD_INITIAL_CLUSTER={{ range $i, $node := .env.metadata.etcd_ignition_nodes }}{{ if $i }},{{end}}{{$node.name}}=http://{{$node.ip}}:2380{{end}}"
             Environment="ETCD_NAME={{.host.name}}"
 
-{{ if index .env "ssh_authorized_keys" }}
 passwd:
   users:
     - name: core
+      password_hash: {{ .boot.metadata.root_pass }}
+      {{- if index .host.metadata "ssh_authorized_keys" }}
```

Comment:
Since we have 3 hosts using this definition lets make this one an `env.metadata.ssh_authorized_keys`